ephios/core/templates/core/userprofile_confirm_password_reset.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}
+    {% translate "Password reset" %}
+{% endblock %}
+
+{% block content %}
+    <div class="page-header">
+        <h1>{% translate "Password reset" %}</h1>
```

Comment:
maybe indicate that this is meant for another user to distinguish from the own password reset?